src/pages/JobOwner/Talent/Index.tsx

```diff
@@ -43,7 +43,7 @@ function Talent() {
     isLoading: isViewLoading,
   } = useGetRecentlyViewedFreelancersQuery();
 
-  const talentPart: ITalentPart[] = [
+  const talentParts: ITalentPart[] = [
```

Comment:
this constant have initialisation with each rerender, its very very bad practice